crates/frontend/src/pages/dimensions.rs

```diff
@@ -64,13 +88,20 @@ pub fn dimensions() -> impl IntoView {
                 open_drawer("dimension_drawer");
             };
 
-            let edit_icon: HtmlElement<html::I> =
-                view! { <i class="ri-pencil-line ri-xl text-blue-500"></i> };
+            let handle_dimension_delete = move |_| {
+                set_delete_id.set(Some(dimension_name.clone()));
+                set_delete_modal_visible.set(true);
+            };
 
             view! {
-                <span class="cursor-pointer" on:click=edit_click_handler>
-                    {edit_icon}
-                </span>
+                <div class="join">
+                    <span class="cursor-pointer" on:click=edit_click_handler>
+                        <i class="ri-pencil-line ri-xl text-blue-500"></i>
+                    </span>
+                    <span class="cursor-pointer text-red-500" on:click=handle_dimension_delete>
```

Comment:
Should we hide the delete button and the edit button for `variantIds`? :thinking:

Reply:
Hiden delete button , let this be editable , once can edit priority and add function